client/packages/requisitions/src/RnRForms/DetailView/ContentArea.tsx

```diff
@@ -84,31 +84,31 @@ export const ContentArea = ({
           '.first-column': {
             left: 0,
             position: '-webkit-sticky',
-            width: 80,
+            minWidth: 80,
           },
           '.second-column': {
-            borderRight: '1px solid blue',
-            left: 79,
+            left: 80,
             minWidth: '300px',
             position: '-webkit-sticky',
           },
         }}
       />
       <Table
         sx={{
-          height: '500px',
```

Comment:
was getting a bit of this 😁 
![Uploading Screenshot 2024-08-19 at 9.19.32 AM.png…]()